manifests/ca.pp

```diff
@@ -26,8 +27,8 @@
   file { $ca_key_password_file:
     ensure    => file,
     content   => $ca_key_password,
-    owner     => 'root',
-    group     => 'root',
+    owner     => $owner,
+    group     => $group,
```

Comment:
This doesn't look right. In the Katello scenario we set the group to foreman (https://github.com/theforeman/foreman-installer/blob/bef5bd620a5b1494d3b5779030bc6b29e861645c/config/katello-answers.yaml#L12-L13) and Foreman doesn't need to read this file.